packages/atlas/src/MainLayout.tsx

```diff
@@ -108,12 +108,14 @@ export const MainLayout: FC = () => {
   return (
     <>
       <CookiePopover />
+      {/*<Suspense fallback={<div>adawd</div>}>*/}
```

Comment:
suspense will be included when it'll be supported by apollo client

Reply:
This should be deleted, since I introduced it in `Viewer` and `Studio` layout